nusight2/src/client/components/visual_mesh/camera/view_model.ts

```diff
@@ -112,30 +110,30 @@ export class CameraViewModel {
 
   private meshGeometry = createTransformer(
     (mesh: VisualMesh): BufferGeometry => {
-      const { rows, indices, neighbours, coordinates, classifications } = mesh
+      const { neighbours, coordinates, classifications } = mesh
 
       const nElem = coordinates.length / 2
 
       // Cumulative sum so we can work out which row our segments are on
-      const cRows = rows.reduce(
-        (acc, v, i) => {
-          acc.push(acc[i] + v)
-          return acc
-        },
-        [0],
-      )
+      // const cRows = rows.reduce(
+      //   (acc, v, i) => {
+      //     acc.push(acc[i] + v)
+      //     return acc
+      //   },
+      //   [0],
+      // )
 
       // Calculate our position
-      const position = ([] as number[]).concat(
-        ...indices.map(i => {
-          // Which ring we are on as a value between 0 and 1
-          const idx = bounds.le(cRows, i)
-          const phi = idx / rows.length
-          // How far around the ring we are as a value between 0 and 1
-          const theta = (i - cRows[idx]) / rows[idx]
-          return [phi, theta]
-        }),
-      )
+      // const position = ([] as number[]).concat(
```

Comment:
Why commented out code everywhere?

Reply:
Because visual mesh message changed and broke nusight and I am ill-equipped to fix this. Also, the visual mesh view on nusight is rarely (if ever) used so I feel zero bads in leaving it broken a little longer